aten/src/THNN/generic/Col2Im.c

```diff
@@ -116,15 +116,15 @@ static inline void THNN_(Col2Im_shapeCheck)(
                          THTensor *input,
                          THTensor *gradOutput,
                          int64_t outputHeight, int64_t outputWidth,
-                         int64_t kH, int64_t kW, int64_t dH, int64_t dW,
-                         int64_t padH, int64_t padW, int64_t sH, int64_t sW) {
+                         int64_t kH, int64_t kW, int64_t dilationH, int64_t dilationW,
+                         int64_t padH, int64_t padW, int64_t dH, int64_t dW) {
```

Comment:
Suggestion for the future: Split the argument renaming into a patch on its own, so it's easy to review.